start/process.go

```diff
@@ -209,3 +210,14 @@ func (p *process) respawn() {
 	p.waitPid()
 	p.output.WriteBoldLinef(p, "Restarted with pid %v...", p.pid)
 }
+
+func (p *process) reportExitCode() {
+	exitCode := p.tmux.ExitCode()
+	message := "Exited"
+
+	if exitCode != 0 {
+		message = fmt.Sprintf("Exited with code %d", exitCode)
+	}
```

Comment:
Does it hurt to always print exit code?

```suggestion
	message := fmt.Sprintf("Exited with code %d", exitCode)
```

Reply:
done!